modules/signals/rxjs-interop/spec/rx-method.spec.ts

```diff
@@ -171,7 +171,7 @@ describe('rxMethod', () => {
     service.method(sig);
     service.method(1);
     flushEffects();
-    expect(results).toEqual([1, 1, 1]);
+    expect(results).toEqual([1, 1]);
```

Comment:
This is something we'll have to take a closer look at.
cc @markostanimirovic

Reply:
This is fixed in da953ab0b9bf3217cae7291b1accb7b6220d5f66 by invoking `detectChanges`, as per https://github.com/angular/angular/blob/main/CHANGELOG.md#core-11